shiny/_app.py

```diff
@@ -89,7 +97,7 @@ def server(input: Inputs, output: Outputs, session: Session):
 
     def __init__(
         self,
-        ui: Tag | TagList | Callable[[Request], Tag | TagList],
+        ui: Tag | TagList | Callable[[Request], Tag | TagList] | Path,
```

Comment:
@jcheng5 What do you think about taking a `Path` object to indicate that it should read the UI from a file?